### DISCUSSION TITLE
Convert presentation to pdf with 16:9 ratio

### DISCUSSION THREAD

**Discussion by Lukabuntu** [senshu/Sozi#675] [Q&A]
Hello everyone,

i would like to ask whether it is possible in Sozi to export the presentation in a format not specified as default. In particular, many presentations nowadays need to be in a 16:9 ratio since beamers work with this. Is it somehow possible to export a 16:9 pdf presentation? Or does anybody know any workaorunds? Thanks for any help.

Best regards

**Comment by aumouvantsillage** [comment]
Unfortunately, when exporting to PDF, the supported aspect ratios are limited to common printed document page sizes.

Sozi supports some of the presets available in [Electron](https://www.electronjs.org/)'s [printToPDF](https://www.electronjs.org/docs/latest/api/web-contents#contentsprinttopdfoptions) function.
I see that printToPDF also supports custom page sizes but this feature is not available from the user interface of Sozi.
